src/components/home/HomePage.scss

```diff
@@ -0,0 +1,351 @@
+#navbar {
+  display: flex;
+  height: 100px;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #6676a0;
+  color: white;
+  justify-content: flex-end;
+  position: relative;
+  padding: 0;
+  overflow-x: hidden;
+
+  .collapseMenuItems {
+    width: 80%;
+    justify-content: flex-end;
+  }
+
+  .navbarItems {
+    width: 50%;
+    padding: 10px 5px 10px 5px;
+
+    > .col {
+      padding-left: 5px;
+      padding-right: 5px;
+      transition: font-size 0.3s ease-in-out;
+    }
+  }
+
+  .navbarLogo {
+    position: absolute;
+    max-width: 100px;
+    left: 100px;
+    cursor: pointer;
+  }
+
+  > div {
```

Comment:
let's add a class to this, in general `> div` is a very fragile selector. what i mean is that if your teammate wants to add another div for some other reason, it will be hard for them